docs/source/benchmark.rst

```diff
@@ -0,0 +1,38 @@
+.. _GPU benchmarks:
+
+==============
+GPU benchmarks
+==============
+
+We have done some benchmarks on our department to find out differences between GPUs and we have
+decided to shared them here. Therefore they do not test speed of NeuralMonkey, but they 
```

Comment:
Jindra myslel "NeuralMonkey" -> "Neural Monkey".
(Já bych zrovna mírně preferoval bez mezery camel-case, aby Jindra je autor a rozhodl se pro mezeru a už je to skoro všude s mezerou, takže nechť je to konzistentní.)